protos/bottle/fulfillment/v1/order.proto

```diff
@@ -5,15 +5,20 @@ package bottle.fulfillment.v1;
 import "bottle/account/v1/address.proto";
 import "bottle/account/v1/user.proto";
 import "bottle/catalog/v1/product.proto";
+import "bottle/fulfillment/v1/line_group.proto";
+import "bottle/fulfillment/v1/tax_line.proto";
 
 message Order {
   string id = 1;
 
   bottle.account.v1.User customer = 2;
   bottle.account.v1.Address shipping = 3;
   bottle.account.v1.Address billing = 4;
+
   string shipping_method = 5;
-  repeated bottle.catalog.v1.Product products = 6;
+  double shipping_price = 10;
+
+  repeated LineGroup line_groups = 11;
```

Comment:
I'm open to better names here. My first thought was `LineItem` but then what do we call the line items under that? I figure `LineGroup` would have many `LineItem`s on it.

Also putting all of the `rush` and `warranty` lines on the `LineGroup` means that when we display to the customer, we need to fake those lines if we want them to show.

Alternatively we could make `rush` and `warranty` `LineItem`s, but then we would want to add some `type` field to make it easier to filter out.

Reply:
> Also I'm not sure that's something we're worrying about today

What line items exist on line items? I'm not sure I follow.